src/langs/en.json

```
@@ -99,5 +100,20 @@
   "EXPORT_LIST_ITEM_CHATS_DESCRIPTION": "Conversations or discussions related to this item will be captured in the export.",
   "EXPORT_LIST_ITEM_ACTIONS_TITLE": "Tracked Actions",
   "EXPORT_LIST_ITEM_ACTIONS_DESCRIPTION": "Actions or activities that have been recorded for this item within different view including Builder, Library, and Player.",
-  "WHAT_TO_EXPORT_TITLE": "What actions does the archive include?"
+  "WHAT_TO_EXPORT_TITLE": "What actions does the archive include?",
+  "Today": "Today",
+  "Yesterday": "Yesterday",
+  "This Week": "This Week",
+  "Last Week": "Last Week",
+  "This Month": "This Month",
+  "Last Month": "Last Month",
+  "RANGE_DATE_PICKER_INPUT_LABEL": "Start date - End date",
+  "GENERAL_STATISTICS_ACTIVITY_CHART": "Activity chart",
+  "MY_STATISTICS": "My statistics",
+  "GENERAL_STATISTIC_ITEMS_CREATED": "Items created",
+  "GENERAL_STATISTIC_LIKED_ITEMS": "Liked items",
+  "GENERAL_STATISTIC_DOWNLOADED_ITEMS": "Downloaded items",
+  "GENERAL_STATISTIC_CHAT_CREATED": "Chat created",
+  "NO_RESULTS_FOUND": "Sorry, No results found!",
```

Would be interesting to show human readable date here.
```suggestion
  "NO_RESULTS_FOUND": "No activity for the time period of {{period}}",
```